spec/miq_ae_event_spec.rb

```diff
@@ -124,6 +124,14 @@
 
           MiqAeEvent.raise_evm_event("vm_create", vm, :vm => vm)
         end
+
+        it "has current user" do
+          User.current_user = user
```

Comment:
@lfu use User.with_user similar to https://github.com/ManageIQ/manageiq/blob/b978b61737bf8bffcca23ab3710500b1f677d4fe/spec/lib/rbac/filterer_spec.rb#L242

This we we dont have it set for the whole thread.